src/nbl/video/COpenGLCommandBuffer.cpp

```diff
@@ -115,8 +115,7 @@ COpenGLCommandBuffer::~COpenGLCommandBuffer()
         if (!IGPUCommandBuffer::canReset())
             return false;
```

Comment:
if we rewrite to a `reset_impl` then all these checks can be common to all backends.

Reply:
Resolved.